src/osltoy/qtutils.h

```diff
@@ -171,7 +171,7 @@ class DoubleSpinBox final : public QDoubleSpinBox {
         double ss = m_fixed_step;
         if (m_variable_step_digits) {
             ss        = m_variable_min_step;
-            double av = fabs(valsize);
+            double av = ::fabs(valsize);
```

Comment:
I think this would be more clear as std::fabs

Reply:
agreed, should I update that in PR?